test/Dialect/Calyx/round-trip.mlir

```diff
@@ -0,0 +1,47 @@
+// RUN: circt-opt %s -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL: component ComponentWithInAndOutPorts (in1: 64, in2: 16) -> (out1: 32, out2: 8) {
+calyx.component @ComponentWithInAndOutPorts (%in1: i64, %in2: i16) -> (%out1: i32, %out2: i8) {
+
+  // CHECK:      cells {
+  // CHECK-NEXT: }
+  // CHECK-NEXT: wires {
+  // CHECK-NEXT: }
+  // CHECK-NEXT: control {
+  // CHECK-NEXT: }
+  calyx.cells {
+  }
+  calyx.wires {
+  }
+  calyx.control {
+  }
+
+// CHECK: }
```

Comment:
nit: is this really important to check? i think it's best to keep minimal and avoid checking things like trailing braces

Reply:
Removed. Thanks.